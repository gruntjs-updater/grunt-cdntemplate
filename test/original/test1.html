<!DOCTYPE html>
<html lang="en" class="devsite">
	<head>
	<link href="https://developers.google.com/+/web/+1button/" rel="canonical">
	<meta charset="utf-8" />
	<link href='<%= bootstrap.css %>' rel='stylesheet' type='text/css'>
	</head>
	<script src="<%=angular.js %>"></script>

	<body>

	</body>

</html>
