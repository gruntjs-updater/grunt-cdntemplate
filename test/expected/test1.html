<!DOCTYPE html>
<html lang="en" class="devsite">
	<head>
	<link href="https://developers.google.com/+/web/+1button/" rel="canonical">
	<meta charset="utf-8" />
	<link href='//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/css/bootstrap.min.css' rel='stylesheet' type='text/css'>
	</head>
	<script src="//ajax.googleapis.com/ajax/libs/angularjs/1.2.26/angular.min.js"></script>

	<body>

	</body>

</html>
